components/eamxx/src/diagnostics/vertical_layer.hpp

```diff
@@ -9,10 +9,10 @@ namespace scream
 {
 
 /*
- * This diagnostic will produce the potential temperature.
+ * This diagnostic will produce the vertical layer height at midpoint.
```

Comment:
The class does much more now. We should write in the comment all 5 possible quantities that it can compute.